sources/model/GGRealTimeMontior.m

```diff
@@ -695,7 +695,19 @@ - (void)sendWatchDriverWithDriverUUID:(nonnull NSString *)uuid
     
     
 }
+- (void)sendCustomerSuccessEventWithCustomerAccessToken:(nonnull NSString *)customerAccessToken
+                                  customerId:(nonnull NSString *)customerId
+                                        completionHandler:(nullable SocketResponseBlock)completionHandler {
+    
+    NSMutableDictionary *params = [NSMutableDictionary dictionaryWithObjectsAndKeys:
+                                   self.developerToken, @"dev_token",
+                                   customerId,@"customer_id",
```

Comment:
the calling method handles nil values (sendCustomerConnectedEventWithCompletionHandler)
in addition, the signature of the method defines "nonnull" on both properties so you'll get compilation warning if you pass nil .